charts/reform-scan-blob-router/values.aat.template.yaml

```diff
@@ -22,6 +23,7 @@ java:
         - storage-account-staging-primary-key
         - storage-account-staging-secondary-key
         - crime-storage-connection-string
+        - pcq-storage-connection-string
```

Comment:
given exists in required environments (including prod) - all good

Reply:
I added it in aat. The storage account doesn't exist in other environments. I'll add the secret to other environments with some value before I merge the PR

Reply:
if values gonna be used in any environment app - they must exist. e.g. storage clients might be ok until we make a call. but building clients might involve some validation. plus pods won't boot up - it'll error saying secret not found

Reply:
story description says SAS token should be used for uploading files

Reply:
I'll update the PR

Reply:
Added pcq storage account name and account keys. Added the secrets in AAT. I'll add the secrets in other environments before merging the PR.
Removed connection string and deleted the secret from aat key vault.

Reply:
As discussed, removed all secrets related to pcq storage account and just added storage url.